src/util.js

```diff
@@ -1,6 +1,16 @@
 var _ = require( "lodash" );
 var semver = require( "semver" );
 
+function checkInfo( info ) {
```

Comment:
instead of writing this function, I think it would be better to check the resulting format of the image tag info in the GetImageInfo instead. That way, the function has a post condition that it was parsed correctly. If it's not, we should raise an error (or return nothing) so the caller can respond appropriately.